src/JustSaying/Messaging/MessageBatchPublisherExtensions.cs

```diff
@@ -1,29 +0,0 @@
-using System.ComponentModel;
-using JustSaying.Models;
-
-namespace JustSaying.Messaging;
-
-[EditorBrowsable(EditorBrowsableState.Never)]
-public static class MessageBatchPublisherExtensions
```

Comment:
Do you _have_ to delete this?

It's a binary breaking change and would otherwise require us to release JustSaying v8, and we've already just done a large breaking change with v7.

Reply:
I've just created this file, but I deleted it.
We don't had this file before